frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -21,6 +21,7 @@ export const errorTrackingQuery = ({
             select: [
                 'any(properties) -- Error',
                 'properties.$exception_type',
+                'sparkline(arrayMap(number -> countEqual(groupArray(toHour(timestamp)), number), range(24))) -- Volume',
```

Comment:
can you ELI5?
the range(24) makes me wonder what this would do at different time of the day (but I can't wrap my head around what it's doing right now so asking rather than figuring out :))